spec/features/maths_and_physics_claim_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.feature "Maths & Physics claims" do
+  [true, false].each do |javascript_enabled|
```

Comment:
Could this also be:`%w[enabled disabled].each do |javascript_enabled|`?

Reply:
We still need a `true` or `false` value for lie 6 (i.e. `js: javascript_enabled`)

Reply:
Ah. I see, thank you.